src/Statistical_Mechanics/3quantumSM.tex

```diff
@@ -0,0 +1,1395 @@
+    This is a really complex calculation for $T\ge T_c$.\\
+    Recall that classically we had $E = N \frac32 k_BT$, so $c_V = \frac32 k_BT$, which was wrong for low temperatures ($c_V\xrightarrow[T\to 0]{}0$ classically). Now quantum mechanics solves the problem and we can correct it. $c_V(T)$ is plotted in Fig. \ref{fig:cv}, where we can see that it is continuous in $T_c$ but not differentiable (it has a cusp). This is again another signal of a phase transition.
+    \begin{figure}[ht]
+        \centering
+        \includegraphics[width = 8cm]{images/cv(T).png}
+        \caption{$c_V(T)$ for Bosons (red) and Fermions (green). In the Bose-Einstein distribution, the cusp at $T_c$ indicate a phase transition.}
+        \label{fig:cv}
+    \end{figure}
+
+    \item Entropy $S$\\
+    $$
+        S = \frac{E-\Omega -\mu N}T = 
+        k_B\left(\frac{E+\frac23 E-N\beta^{-1}\log Z}{k_BT}\right) =
+        k_BN\left(\frac53\frac1{k_BT}\frac EN -\log Z\right)
+    $$
+    Since $z = e^{\beta\mu}\qquad -pV= \Omega = -\frac23 E$.
+
+    \begin{align*}
+        s &= \frac SN = k_B\left[\frac53\frac{E/N}{k_BT} - \log z\right]\\
+        &= k_B\left(\frac52\frac{g_{5/2}(z)}{g_{3/2}(z)} - \frac{g_{3/2}(z)}{g_{3/2}(z)}\log z\right)\\
+        &= \frac {k_B}{g_{3/2}(z)}\left[\frac52g_{5/2}(z) - g_{3/2}(z)\log z\right]\\
+        &= \frac{k_B}{g_{3/2}} \left(\frac T{T_c}\right)^{3/2}\left(\frac52g_{5/2}(z) - g_{3/2}(z)\log z\right)
+    \end{align*}
+    We can check that this equation holds $\forall T$ as long as we use $g_l(z) = g_l(1)$ for $T \le T_c$:
+    \begin{itemize}
+        \item $T>T_c \qquad z<1\qquad$ all good
+        \item $T=T_c \qquad s(T=T_c) = \frac52 k_B\frac{g_{5/2}(1)}{g_{3/2}(1)}$
+        \item $T<T_c \qquad s = \frac52 k_B\left(\frac T{T_c}\right)^{3/2}\frac{g_{5/2}(1)}{g_{3/2}(1)} \xrightarrow[T\to 0]{}0$\\
+        however, at $T=0$, all the particles are in the ground state ($n_0(T) \xrightarrow[T\to0]{}n$. This means that particles in the condensate carry no entropy $S_{cond} = 0$. So $s$ below $T_c$ gets smaller and smaller as $T$ decreases, since more and more particles go to the ground state.
+    \end{itemize}
+        
+    Consider one particle jumping from $\epsilon > 0$ to the ground state at $T=T_c$. Then:
+    $$ \Delta s = s(T=T_c) - 0 = k_B\frac 53 \frac{g_{5/2}(1)}{g_{3/2}(1)} \ne 0$$
+    But we also know that $0 \ne \Delta S = \Delta Q / T_c$. So the process of of decaying from $\epsilon > 0$ to $\epsilon =0$ costs some energy, some heat. This heat $\Delta Q \ne 0$ is called latent heat.
+
+    This is a peculiar phase transition, with continuous thermodynamic potentials, but latent heat as (for instance) in evaporation.
+
+    \gray{It is the same as water boiling, which stops at $T= 100^oC$ since particles absorb latent heat.}
+\end{itemize}
+
+\notedate{(XII) MON (ex.3) 12/12/2022}
+\notedate{(XIII) MON (ex.4) 19/12/2022}
+\subsection{Exercise (2.5): Gas of photons}
```

Comment:
I guess this should not be empty... in general it's a superb work, well done!
If you don't have this part just tell me and I will merge the PR.